pkg/mock/mockcluster/mockcluster.go

```diff
@@ -36,31 +36,36 @@ import (
 type Cluster struct {
 	*core.BasicCluster
 	*mockid.IDAllocator
-	*mockoption.ScheduleOptions
 	*placement.RuleManager
 	*statistics.HotCache
 	*statistics.StoresStats
+	*config.PersistOptions
 	ID               uint64
 	suspectRegions   map[uint64]struct{}
 	disabledFeatures map[versioninfo.Feature]struct{}
 }
 
 // NewCluster creates a new Cluster
-func NewCluster(opt *mockoption.ScheduleOptions) *Cluster {
+func NewCluster(opts *config.PersistOptions) *Cluster {
 	ruleManager := placement.NewRuleManager(core.NewStorage(kv.NewMemoryKV()))
-	ruleManager.Initialize(opt.MaxReplicas, opt.GetLocationLabels())
+	ruleManager.Initialize(int(opts.GetReplicationConfig().MaxReplicas), opts.GetReplicationConfig().LocationLabels)
 	return &Cluster{
 		BasicCluster:     core.NewBasicCluster(),
 		IDAllocator:      mockid.NewIDAllocator(),
-		ScheduleOptions:  opt,
 		RuleManager:      ruleManager,
 		HotCache:         statistics.NewHotCache(),
 		StoresStats:      statistics.NewStoresStats(),
-		suspectRegions:   make(map[uint64]struct{}),
+		PersistOptions:   opts,
+		suspectRegions:   map[uint64]struct{}{},
 		disabledFeatures: make(map[versioninfo.Feature]struct{}),
 	}
 }
 
+// GetOpts returns the cluster configuration.
+func (mc *Cluster) GetOpts() *config.PersistOptions {
```

Comment:
I think it is not a good idea to use abbreviations in function names.

Reply:
In my next PR `GetOpts` will be used many many many times like this `cluster.GetOpts().GetLeaderScheduleLimit()`, IMO make it shorter can ease some pain later.

Reply:
Why not use `cluster.GetLeaderScheduleLimit()` directly?

Reply:
Reverted. I'm going to rethink it again.